feature/user_profile/src/main/java/org/mifos/mobile/feature/user_profile/screens/UserProfileScreen.kt

```diff
@@ -0,0 +1,295 @@
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import androidx.core.content.ContextCompat
+import androidx.core.graphics.drawable.toBitmap
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import org.mifos.mobile.core.common.Network
+import org.mifos.mobile.core.common.utils.DateHelper
+import org.mifos.mobile.core.datastore.PreferencesHelper
+import org.mifos.mobile.core.model.entity.client.Client
+import org.mifos.mobile.core.model.entity.client.Group
+import org.mifos.mobile.core.ui.component.MFScaffold
+import org.mifos.mobile.core.ui.component.MifosErrorComponent
+import org.mifos.mobile.core.ui.component.MifosProgressIndicatorOverlay
+import org.mifos.mobile.core.ui.component.MifosUserImage
+import org.mifos.mobile.core.ui.component.UserProfileField
+import org.mifos.mobile.core.ui.component.UserProfileTopBar
+import org.mifos.mobile.feature.third.party.user_profile.R
+import org.mifos.mobile.feature.user_profile.utils.TextDrawable
+import org.mifos.mobile.feature.user_profile.utils.UserDetailState
+import org.mifos.mobile.feature.user_profile.utils.UserDetails
+import org.mifos.mobile.feature.user_profile.viewmodel.UserDetailViewModel
+
+/**
+ * @author pratyush
+ * @since 20/12/2023
+ */
+
+@Composable
+fun UserProfileScreen(
+    viewModel: UserDetailViewModel = hiltViewModel(),
+    navigateBack: () -> Unit,
+    changePassword: () -> Unit,
+    client: Client?,
+    bitmap: Bitmap?
+) {
+
+    val uiState by viewModel.userDetailUiState.collectAsStateWithLifecycle()
+    val context = LocalContext.current
+
+    LaunchedEffect(key1 = true) {
+        viewModel.loadUserData()
+    }
+
+    UserProfileScreen(
+        navigateBack = navigateBack,
+        uiState = uiState,
+        onRetry = {
+            viewModel.loadUserData()
+        },
+        changePassword = changePassword,
+        client = client,
+        bitmap = bitmap
+    )
+}
+
+@Composable
+fun UserProfileScreen(
+    navigateBack: () -> Unit,
+    uiState: UserDetailState,
+    onRetry: () -> Unit,
+    changePassword: () -> Unit,
+    client: Client?,
+    bitmap: Bitmap?
+) {
+    val context = LocalContext.current
+    val snackBarHostState = remember {
+        SnackbarHostState()
+    }
+
+    var newClient = client
+    var newBitmap  = bitmap
+
+    Column( Modifier.fillMaxSize() ){
+
+        UserProfileTopBar(home = navigateBack, text = R.string.user_details)
```

Comment:
can we use scaffold?

Reply:
this screen has custom topBar, that's y i am using the column and custom top bar